XamlControlsGallery/ControlPages/ProgressRingPage.xaml

```diff
@@ -37,7 +34,21 @@
             <local:ControlExample.Substitutions>
                 <local:ControlExampleSubstitution Key="IsActive" Value="{x:Bind ProgressToggle.IsOn, Mode=OneWay}" />
             </local:ControlExample.Substitutions>
-
+        </local:ControlExample>
+        <local:ControlExample HeaderText="A determinate progress ring.">
+            <StackPanel x:Name="Control2" Orientation="Horizontal">
+                <muxc:ProgressRing Width="60" x:Name="ProgressRing2" IsIndeterminate="False"/>
+                <TextBlock x:Name="Control2Output" Style="{ThemeResource OutputTextBlockStyle}" Width="60" TextAlignment="Center" />
+                <muxc:NumberBox x:Name="ProgressValue" AutomationProperties.LabeledBy="{Binding ElementName=ProgressLabel}" Minimum="0" Maximum="100" SpinButtonPlacementMode="Inline" Header="Progress" ValueChanged="ProgressValue_ValueChanged" VerticalAlignment="Center"/>
```

Comment:
There's a known NumberBox bug that it's not announcing its header as its UAI name. The LabeledBy bit would have gotten around that bug...but you got rid of ProgressLabel on my recommendation. So I suggest 

```suggestion
                <muxc:NumberBox x:Name="ProgressValue" AutomationProperties.Name="Progress amount" Minimum="0" Maximum="100" SpinButtonPlacementMode="Inline" Header="Progress" ValueChanged="ProgressValue_ValueChanged" VerticalAlignment="Center"/>
```